apps/dapp-console/app/mixpanel/mixpanel.tsx

```diff
@@ -0,0 +1,14 @@
+import mixpanelBrowser from 'mixpanel-browser'
+
+const initMixpanel = () => {
+  if (
+    process.env.VITE_MIXPANEL_TOKEN === 'false' ||
+    process.env.VITE_MIXPANEL_TOKEN == null
+  ) {
+    return null
+  }
+  mixpanelBrowser.init(process.env.VITE_MIXPANEL_TOKEN)
+  return mixpanelBrowser
+}
+
+export const mixpanel = initMixpanel()
```

Comment:
not sure how you're going to implement the tracking portion, but i think we should avoid using this directly in our component files like we do in gateway meaning this

```
mixpanel?.track(TRACKING_EVENT_NAME.PRETEND_EVENT, 
  [MIXPANEL_CUSTOM_TRACKING_PROPERTY.PRETEND]: 'pretend',
})
```

having to import the custom tracking properties especially if we ever want to use the same event track call in multiple places and having mixpanel across the codebase would make it harder to refactor if we ever dropped mixpanel

instead i think we should wrap all them with functions and keep them under this directory so we can have functions that do this

```
function trackPretendEvent(args: PretendEventArgs) {
    mixpanel?.track('pretendEvent', 
     'Pretend': args.pretendProperty,
   })
}
```

Reply:
oh cool, so the usage would be like 

```
import { trackPretendEvent } from 'event-tracking/mixpanel'

// on the action to track 
trackPretendEvent(relevantArgs)
```

and for each new event name there would be a new event function added to mixpanel.ts taking in the relevant arguments

Reply:
Yup exactly and if the file ever gets too long can be broken out into multiple